src/difference/solution-class-sunday/difference.spec.js

```diff
@@ -0,0 +1,70 @@
+import { difference } from './difference.js';
+
+describe('checks arrays of different elements', () => {
+  it('arrays of different strings', () => {
+    const actual = difference(['hello', 'thank you', 'bye'], ['hello', 'bye']);
+    expect(actual).toEqual(['thank you']);
+  });
+  it('arrays of different numbers', () => {
+    const actual = difference([3, 5, 7, 9], [5, 7, 9, 11]);
+    expect(actual).toEqual([3]);
+  });
+  it('mixed arrays of strings, numbers, booleans', () => {
+    const actual = difference(
+      ['hello', true, false, 0, 1, 2],
+      ['hello', false, 1, 2],
+    );
+    expect(actual).toEqual([true, 0]);
+  });
+});
+
+describe('checks arrays which include repeating elements', () => {
```

Comment:
I think we should remove duplicated values in the returned array like in the example in line 17 src/difference/solution-class-sunday/difference.js

Reply:
all tests are working! and it was necessary to check what will happen with such arrays that include duplicated values

Reply:
Yes but

```
**Example**

```js
difference([1, 2, 1], [2, 3]); //  [1]
```
like in the example it should return [1] not [1,1]
for this that link can be helpful for you 

Removing duplicates;
https://levelup.gitconnected.com/7-ways-to-remove-duplicates-from-array-in-javascript-cea4144caf31

 if you wish to change; otherwise, everything others seem good and clear, readable. If you would like to continue with other exercises I will merge this. Please let me know. Good luck.